webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js

```diff
@@ -182,29 +182,34 @@ $.extend(Timepicker.prototype, {
 		tp_inst.ampm = '';
 		tp_inst.$input = $input;
 
-		if (o.altField)
+		if (o.altField) {
 			tp_inst.$altInput = $(o.altField)
 				.css({ cursor: 'pointer' })
 				.focus(function(){ $input.trigger("focus"); });
+		}
 		
 		if(tp_inst._defaults.minDate==0 || tp_inst._defaults.minDateTime==0)
 		{
 			tp_inst._defaults.minDate=new Date();
 		}
-		if(tp_inst._defaults.maxDate==0 || tp_inst._defaults.maxDateTime==0)
+		if(tp_inst._defaults.maxDate===0 || tp_inst._defaults.maxDateTime===0)
 		{
 			tp_inst._defaults.maxDate=new Date();
 		}
 		
 		// datepicker needs minDate/maxDate, timepicker needs minDateTime/maxDateTime..
-		if(tp_inst._defaults.minDate !== undefined && tp_inst._defaults.minDate instanceof Date)
+		if(tp_inst._defaults.minDate !== undefined && tp_inst._defaults.minDate instanceof Date) {
```

Comment:
What do you think about,
```suggestion
		if(tp_inst._defaults.minDate && tp_inst._defaults.minDate instanceof Date) {
```

Reply:
Hi, don't know how this happened ill fix it just now thanks for telling me 😄